cli/portforward_test.go

```diff
@@ -21,10 +21,8 @@ import (
 	"github.com/coder/coder/testutil"
 )
 
+//nolint:tparallel,paralleltest // Subtests require setup that must not be done in parallel.
```

Comment:
I'm hoping removing parallel from the parent will allow these to run more stable, so for now let's re-enable these tests.